### ISSUE TITLE
Test steps hidden in test body

### ISSUE THREAD

**Issue by volkanogretmen** [badeball/cypress-cucumber-preprocessor#796]
I've been using this plugin in the past and I noticed that the "Given, When, Then" steps are now hidden within the test body, which makes reading/debugging tests a hard task.

Is there a possibility to add an option to move the actual test steps 1 scope higher, so instead of rendering it inside the test body, have each "Given, When, Then" step it's own `it` step as we are used to in Cypress and also in the previous version (so that way it's also collapsible in the runner)?

Example scope:
- Feature: => describe()
- Scenario => context()
- Given,When,Then => it()
- Test steps => test body 

![image](https://user-images.githubusercontent.com/25029377/183836590-8b1e0d6b-0257-418c-b13f-99abe4acba1a.png)

**Comment by badeball** [comment]
I don’t think you understand the implications of what you’re asking. “Given” describes a precondition to a scenario, but all state is reset between each “it”. Do you see how ill thought out this is?

**Comment by volkanogretmen** [comment]
I just wished it was more readable, that's all.

I found an old repo (with Cypress 4.x) and they solved this issue back then by highlighting the Given,When,Then steps in bold. Perhaps that's something we could consider?

![image](https://user-images.githubusercontent.com/25029377/183841862-25b331f6-adc0-4bd4-8274-5fc7893537a2.png)

**Comment by badeball** [comment]
As far as visibility goes, I agree it's less-than-ideal right now. I do think one can make stuff bold. I'll look into it more when I have the time.

**Comment by badeball** [comment]
This might be relevant for anyone wanting to take a stab at this: https://j1000.github.io/blog/2022/10/27/enhanced_cypress_logging.html (from MetaWhirledPeas at Discord).

**Comment by vojtech-cerveny** [comment]
Just small thing: We have patch for that - We have only emoji before keyword, maybe this can just easy-fix workaround
```
--- a/lib/create-tests.js
+++ b/lib/create-tests.js
@@ -297,9 +297,15 @@ function createPickle(context, scenario, pickle) {
                 cy.then(() => {
                     Cypress.log({
                         name: "step",
-                        displayName: (0, assertions_1.assertAndReturn)("keyword" in scenarioStep && scenarioStep.keyword, "Expected to find a keyword in the scenario step"),
+                        displayName: "▶️ " + (0, assertions_1.assertAndReturn)("keyword" in scenarioStep && scenarioStep.keyword, "Expected to find a keyword in the scenario step"),
                         message: text,
                     });
```

**Comment by badeball** [comment]
I've introduced log groups as seen in the above-mentioned blog post, released as v15.1.0

**Comment by iomedico-beyer** [comment]
Wow, this is so much better. Thank you! I just love it. Do you think you could also put everything ``Before`` into a log group?
